lib/redmine/menu_manager/menu_helper.rb

```diff
@@ -77,10 +77,20 @@ def display_main_menu?(project)
 
   def render_menu(menu, project = nil)
     links = []
-    menu_items_for(menu, project) do |node|
+    classes = ''
+    menu_items = menu_items_for(menu, project) do |node|
       links << render_menu_node(node, project)
     end
-    links.empty? ? nil : content_tag('ul', links.join("\n").html_safe, class: 'menu_root')
+    if menu == :project_menu
+      selected = any_item_selected?(select_leafs(menu_items))
+      classes << (selected ? 'open' : 'closed')
+    end
+
+    links.empty? ? nil : content_tag('ul', links.join("\n").html_safe, class: 'menu_root ' + classes)
```

Comment:
If possible, use `safe_join(links, "\n")` instead to avoid forcing them to be html_safe